spec/models/technical_metadata.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+# rubocop:disable RSpec/FilePath
```

Comment:
Is there a reason why this is here? Why can't you just rename the file to `technical_metadata_spec.rb` ?